app/view/show.py

```diff
@@ -17,9 +17,20 @@
 from collections import defaultdict, OrderedDict
 from jinja2.utils import escape
 
+def get_bug_project(pkgs):
```

Comment:
I may be too picky, but I don't think this method name is clear enough. Maybe we should rename it to `get_package_bug_categories()` or so?